CMakeLists.txt

```diff
@@ -235,6 +235,17 @@ if (NANOGUI_BUILD_SHARED)
   target_link_libraries(nanogui ${NANOGUI_EXTRA_LIBS})
 endif()
 
+install(
+  TARGETS nanogui
+  LIBRARY DESTINATION lib
+  ARCHIVE DESTINATION lib
+  )
+
+install(
+  DIRECTORY include/nanogui DESTINATION include
```

Comment:
It is the latter version. The folder `nanogui` will be installed into the `DESTINATION/include` folder, so that you can include like this `#include<nanogui/nanogui.h>`